Android.mk

```
@@ -1,6 +1,16 @@
 LOCAL_PATH:= $(call my-dir)
 
+ifneq (,$(wildcard frameworks/av/media/mediaserver/Android.mk))
 DROIDMEDIA_32 := $(shell cat frameworks/av/media/mediaserver/Android.mk |grep "LOCAL_32_BIT_ONLY[[:space:]]*:=[[:space:]]*" |grep -o "true\|1\|false\|0")
+else
+ifneq (,$(wildcard frameworks/av/media/mediaserver/Android.bp))
+ifneq (,$(shell cat frameworks/av/media/mediaserver/Android.bp |grep "compile_multilib:[[:space:]]*\".*32\""))
```

nitpick: why would the current grep also match something like `compile_multilib: "abc32"`? May I suggest   [Halium](https://github.com/Halium/libhybris/commit/a6393e3ce3376bcaa18fcfb7e63a0df8c6f4ffea)'s approach?
```suggestion
ifneq (,$(shell cat frameworks/av/media/mediaserver/Android.bp | grep compile_multilib | grep -w "32"))
```